src/PlatformServices/Deployment/TestRunDirectories.cs

```diff
@@ -61,6 +61,9 @@ public string OutDirectory
             get
             {
                 return Path.Combine(this.RootDeploymentDirectory, DeploymentOutDirectorySuffix);
+
+                // Previous corelib impl was:
```

Comment:
Is it still needed? If so, could we explain why we changed the logic?

Reply:
Those comments are grandfathered from MSTest v2 - I'll check if it breaks anything and remove.